climpred_skeleton/core.py

```diff
@@ -0,0 +1,28 @@
+import xarray as xr
+
+
+class Verification:
+    """Absolute base class. Begins the pipeline of verifying a forecast.
+
+    Holds a prediction that is being verified against observations. Stores
+    helpful base attributes (`nmember`, etc.) and runs checks on input
+    data.
+    """
+
+    def __init__(self, prediction: xr.Dataset, observation: xr.Dataset):
```

Comment:
you want this explicitly for xr.Datasets and not dataarrays?

Reply:
also I wouldn't call it prediction, rather `initialized`. this is true for the PM and hindcast setup.